.connect.yml

```diff
@@ -0,0 +1,16 @@
+default:
```

Comment:
What is the difference between this file and `.internal.yml`? Are they both necessary?

Reply:
Yeaaah. `.internal.yml` is what the QuickStart is using at the moment. Obviously both are configurable, but there are some minor differences that are not nailed down until we have a broader discussion as a team, and I would love to wait to eat that change until the change is nailed down. If you don't mind having both for the short term, it would make my life easier :smile:

Reply:
I don't mind having either files. this was mostly a curious question. If `.internal.yml` is being used for the QuickStart, then what is `.connect.yml` is being used for?

Reply:
That's the `connectapi` package (R package to simplify interacting with the Connect Server API). That's the experimental one that I want to chat with y'all about. I.e. the idea is that `.connect.yml` (or some other such thing) would replace `.internal.yml`.

Reply:
Got it, thank you for clarifying!